src/components/configure/ExportImport.jsx

```diff
@@ -0,0 +1,129 @@
+                const jsonObj = JSON.parse(getdata);
+                dispatch(increment("pendingRequests"))
+                client.projects.importFiles(projectID, { secret: jsonObj.secret, id: projectID, name: projectName, modules: jsonObj.modules })
+                    .then(() => {
+                        setProjectConfig(projects, projectID, "modules", jsonObj.modules);
+                        setProjectConfig(projects, projectID, "secret", jsonObj.secret)
+                        notify("success", "Success", "File uploaded successfully")
+                    })
+                    .catch(ex => notify("error", "Error", ex))
+                    .finally(() => dispatch(decrement("pendingRequests")))
+                setType("");
+            }
+            if (Type === "yaml" && extension === "yaml") {
+                const yamlObj = yaml.safeLoad(getdata);
+                dispatch(increment("pendingRequests"))
+                client.projects.importFiles(projectID, { secret: yamlObj.secret, id: projectID, name: projectName, modules: yamlObj.modules })
+                    .then(() => {
+                        setProjectConfig(projects, projectID, "modules", yamlObj.modules);
+                        setProjectConfig(projects, projectID, "secret", yamlObj.secret)
+                        notify("success", "Success", "File uploaded successfully")
+                    })
+                    .catch(ex => notify("error", "Error", ex))
+                    .finally(() => dispatch(decrement("pendingRequests")))
+                setType("");
+            }
+        };
+
+        reader.onerror = function () {
+            console.log(reader.error);
+        };
+        reader.readAsBinaryString(file);
+
+    };
+
+
+    return (
+        <div>
+            <p>Download project config in JSON/YAML file </p>
+            <Form layout="inline">
+                <Form.Item>
+                    <Button onClick={downLoadYaml}>
+                        Yaml
+                        <Icon type="download" />
+                    </Button>
+                </Form.Item>
+                <Form.Item>
+                    <Button onClick={downLoadJson}>
+                        Json
+                        <Icon type="download" />
+                    </Button>
+                </Form.Item>
+            </Form>
+            <p>Update project config by importing a JSON/YAML files</p>
+            <Form layout="inline">
+                <Form.Item>
+                    <Upload onChange={importFiles}>
+                        <Button onClick={() => setType('yaml')}>
+                            <Icon type="upload" /> Yaml
+                    </Button>
+                    </Upload>
+                </Form.Item>
+                <Form.Item>
+                    <Upload onChange={importFiles}>
+                        <Button onClick={() => setType('json')}>
+                            <Icon type="upload" /> Json
```

Comment:
Your upload is not restricting the type of files that can be uploaded and is also making a default POST request.

Example of how I used ant's upload in my previous apps:
```jsx
<Upload accept=".json"
    beforeUpload={file => false}>
      <Button><Icon type="upload" />JSON</Button>
</Upload>
```